lib/internal/cluster/child.js

```diff
@@ -99,7 +99,7 @@ function shared(message, handle, indexesKey, cb) {
     delete handles[key];
     delete indexes[indexesKey];
     return close.apply(this, arguments);
-  };
+  }.bind(handle);
```

Comment:
Mind explaining this one? Typically `bind` isn't that great at not causing deopts :D